Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Valeri Karpov <valkar207@gmail.com>",
            "Saghm Rossi <saghmrossi@gmail.com>",
            "Kevin Yeh <kevinyeah@utexas.edu>"]
 
-description = "An experimental MongoDB driver written by MongoDB interns."
+description = "A native-Rust MongoDB driver. Originally written by MongoDB interns, the driver is an unofficial but still actively maintained project."
```

Comment:
I think the main points that we're circling around is that:
1. It's associated with/developed under MongoDB, i.e. it's under MongoDB "incubation" via Labs
2. It's not officially supported by MongoDB, which can either mean it's maintained by the "community" (not necessarily) or that it's not production-approved (more closely aligned to what we want to get across).

I don't think there's a reason to differentiate "[originally] developed under" and "maintained by"; I think both of these are underscored by its position as a `mongodb-labs` project, and there's not much of a reason to say that it's "actively maintained" — people can tell the status of a project by it's commit and release history. In the end, I agree that something close to our first two interpretations would be good, but that the semantics can be summarized more succinctly by the connotation of it being a "labs project".

"A native MongoDB driver developed under MongoDB labs."

/shrug